src/style.css

```diff
@@ -1,3 +1,80 @@
+:root {
+  --color__nav: #00acdc;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.logo {
+  margin-left: 50px;
+}
+
+.logo__link {
+  width: 40px;
+  height: 40px;
+  vertical-align: bottom;
+  display: block;
+}
+
+.header__content {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.container {
+  max-width: 1200px;
+  min-width: 1024px;
+  margin: 0 auto;
+}
+
+.nav {
+  margin-right: 50px;
+  justify-content: space-around;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style: none;
+  justify-content: space-around;
+}
+
+.nav__link.is-active {
```

Comment:
<img width="952" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/86789161/01db72ef-883a-4ede-bb3f-fcc695dcbe87">
And in all other places